src/shared/test/java/com/couchbase/lite/internal/ExecutionServicesTest.kt

```diff
@@ -260,8 +259,8 @@ class ExecutionServicesTest {
         try { assertTrue(finishLatch.await(5, TimeUnit.SECONDS)) }
         catch (ignore: InterruptedException) { }
 
-        // within 10% is good enough
-        assertEquals((t - 777) / 10, 0L)
+        // within ~10% is good enough
+        assertEquals(0L, (t - 777) / 80)
```

Comment:
Yeah, yeah.... that's totally what I meant too say!

I'm going to suggest that you do this (or, something better!):
```
DELAY = 1500

...
 executionService.postDelayedOnExecutor(
        executionService.postDelayedOnExecutor(
                DELAY,
...
assertEquals(0L, (t - DELAY) / (DELAY / 10))
```